bagitobjecttransfer/recordtransfer/forms/user_forms.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any
+
+from django import forms
+from django.contrib.auth import password_validation
+from django.utils.translation import ugettext_lazy as _
+
+from recordtransfer.models import User
+
+
+class UserProfileForm(forms.ModelForm):
+    """Form for editing user profile."""
+
+    gets_notification_emails = forms.BooleanField(
+        required=False, label=_("Receive notification emails?")
+    )
+    current_password = forms.CharField(
+        widget=forms.PasswordInput, label=_("Current Password"), required=False
+    )
+    new_password = forms.CharField(
+        widget=forms.PasswordInput,
+        label=_("New Password"),
+        required=False,
+        validators=[password_validation.validate_password],
+    )
+    confirm_new_password = forms.CharField(
+        widget=forms.PasswordInput, label=_("Confirm New Password"), required=False
+    )
+
+    class Meta:
+        """Meta class for UserProfileForm."""
+
+        model = User
+        fields = ("gets_notification_emails",)
```

Comment:
Add current_password, new_password, and confirm_new_password to `fields`

Reply:
From reading about [Meta's field attribute](https://docs.djangoproject.com/en/5.1/topics/forms/modelforms/#selecting-the-fields-to-use), it seems that this is used to select fields **from the specified Model** that we wish to expose to the user so that their values can be modified.

Since `current_password`, `new_password`, and `confirm_new_password` are not part of the User model (perhaps the current User password is but I think Django offers functions like `User.check_password` and `User.set_password` to deal with verification and changing of password), I think the way I'm defining fields for these outside of Meta is appropriate.

Reply:
Good catch, yes, that makes sense. I forgot that that's what Meta was for, it's been a little while since i made a form.